.circleci/config.yml

```diff
@@ -49,22 +49,20 @@ notify-on-fail: &notify-on-fail
 
 version: 2.1
 orbs:
-  node: circleci/node@1.1.6
+  node: circleci/node@4.1.0
   slack: circleci/slack@4.1.1
 jobs:
   build-and-test:
-    executor:
-      name: node/default
+    executor: node/default
     steps:
       - checkout
-      - node/with-cache:
-          cache-version: v2
-          steps:
-            - run: npm install
-            - run: npm run init
-            - run: npm run lint
-            - run: npm run build # somehow linting rules differ from eslint
-            - *notify-on-fail
+      - node/install-packages
+      - run: npm run init
+      - run: npm run prettier:check
+      - run: npm run eslint:src
+      - run: npm run test
+#      - run: npm run build # is there any point in building here?
```

Comment:
We can probably just delete this line. We build later before deploying hosting anyway.

Reply:
Yep seems like we can given it was last so nothing depends on it